awx/ui/test/e2e/tests/test-jobs-portal-list-actions.js

```diff
@@ -7,6 +7,10 @@ let data;
 
 module.exports = {
     before: (client, done) => {
+        client.resizeWindow(1200, 800);
```

Comment:
Moving this block of code here makes the login happen while the resources are bring created.